nc-news-react/src/components/Comments.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import { getCommentsByArticleId } from "../axios"
+
+function Comments () {
+
+    const [comments, setComments] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+    const [err, setErr] = useState(false)
+    const {article_id} = useParams()
+
+
+    useEffect(()=> {
+        getCommentsByArticleId(article_id).then((commentsData)=> {
+            setComments(commentsData)
+            setIsLoading(false)
+        }).catch((err)=> {
+            setErr(true)
+        })
+    })
+
+    console.log(comments)
+
+if(isLoading){return <p>Loading...</p>}
+
+    return (
+        <div className="commentsList">
+        {comments.map((comment)=> {
+            return <div className="commentsCard">
+                <p>Author: {comment.author}</p>
+                <p>Comment: {comment.body}</p>
+             </div>
+  })}
+        </div>
```

Comment:
If I go to an article that has no comments, like article 37, I just get a blank page. Would be nice to give the user some feedback that there are no comments currently